server/app/models/sync.rb

```diff
@@ -94,7 +98,17 @@ def set_defaults
     self.status ||= self.class.aasm.initial_state.to_s
   end
 
+  def interval_schedule?
```

Comment:
these methods will be automatically created for enums. No need to add these methods ?

Reply:
interval_schedule? method interval_schedule? checks whether the schedule_type is equal to "interval" and returns a boolean value accordingly.
 any suggestion for this?